firmware/ec/src/devices/at45db.c

```diff
@@ -0,0 +1,213 @@
+/**
+* Copyright (c) 2017-present, Facebook, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree. An additional grant
+* of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#include "inc/devices/at45db.h"
+#include "inc/common/spibus.h"
+#include "inc/common/global_header.h"
+#include <inc/global/OC_CONNECT1.h>
+
+#define AT45DB_MANFACTURE_ID            0x1F
+#define AT45DB_DEVICE_ID                0x0028
+#define AT45DB_READY                    0x80 /* AT45DB Ready Value */
```

Comment:
Isnt AT45DB_READY also an OPCODE  ? if yes please make it AT45DB_READY_OPCODE

Reply:
this is not opcode, just at45db status value.

Reply:
As per our current coding guidelines the "#include"s are to be in alphabetical order . If the same needs to be extended for "#define"s as well then we need to change this as part of a separate request as a lot of files might need to be changed .